tests/e2e/tests/pilot/testdata/v1alpha3/external-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: ExternalService
+metadata:
+  name: external-service-bookinfo
+spec:
+  hosts:
+    - eu.bookinfo.com
+  ports:
+  - number: 80
+    name: http
+    protocol: HTTP
+  discovery: STATIC
+  endpoints:
+  - address: t.istio-system.svc.cluster.local # TODO: this is brittle
```

Comment:
what is the plan here? you are creating an external service that is an alias for `t`, the no proxy app?

Reply:
yep